manifest.json

```
@@ -6,7 +6,7 @@
         "en": "Search engine for PeerTube videos and channels",
         "fr": "Moteur de recherche de vidéos et chaînes PeerTube"
     },
-    "version": "0.0.1.2022.06.03~ynh1",
+    "version": "0.0.1.2023-01-04~ynh1",
```

```suggestion
    "version": "0.0.1.2023.01.04~ynh1",
```

(can't use `-` in version number, we shall stick on `a.b.c.x.y.z` syntax, otherwise it's not clear wether `2022.06.03` is greater or lower than `2023-01-04`)

---

Thanks for the hint!